internal/pkg/table/path.go

```diff
@@ -36,21 +37,31 @@ const (
 
 type Bitmap struct {
 	bitmap []uint64
+	m      *sync.Mutex
```

Comment:
This should be the zero value
```suggestion
	m      sync.Mutex
```